helpers/constants.js

```diff
@@ -83,17 +70,10 @@ var constants = {
 		dappDeposit: 10000000,
 		data: 10000000,
 	},
-	feeStart: 1,
-	feeStartVolume: 10000 * 100000000,
-	fixedPoint: Math.pow(10, 8),
```

Comment:
`fixedPoint` was used and should be used in here https://github.com/LiskHQ/lisk/blob/02d35b04afb2384db1e1b4257b0b8d202ea3fda5/schema/swagger.yml#L1102 . Also `lisk-js` uses it in their test suites as a hardcoded value. We need to ping @Tosch110 @shuse2 & @willclarktech once it is settled, suggesting to integrate it as a constant.